solutions/functions/pivot-index.js

```diff
@@ -0,0 +1,42 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+
+function pivotIndex(arr){
+    // Implement this function
+    let leftSum = 0;
+    let rightSum = 0;
+    for(let i = 0; i < arr.length; i++){
+        rightSum += arr[i];
+    }
+
+    let result = -1;
+    for(let i = 0; i < arr.length; i++){
```

Comment:
@Abduliitr Rename `i` to `pivotIndex`

Reply:
`pivotIndex` is also the name of the function. Will that be okay?

Reply:
I see. `i` is ok in that case.